packages/webapp/src/components/Filter/FilterMultiSelect/index.tsx

```diff
@@ -47,22 +58,25 @@ export const FilterMultiSelect = ({
 
   return (
     <ReactSelect
-      style={style}
-      placeholder={`${t('common:SELECT')}...`}
+      //@ts-ignore
+      style={style} // I suspect the forwardRef is at fault for the type error?
```

Comment:
May as well type that one out too? lol 

https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/forward_and_create_ref/

Reply:
Noooooo not the React Select 🤣 I really didn't want to touch that, but I guess it's imported a lot; Nav had issues on his component too. Thanks for link.